ompi/mca/osc/ucx/osc_ucx_component.c

```diff
@@ -180,6 +181,12 @@ static int component_register(void) {
     (void) mca_base_component_var_register(&mca_osc_ucx_component.super.osc_version, "acc_single_intrinsic",
                                            description_str, MCA_BASE_VAR_TYPE_BOOL, NULL, 0, 0, OPAL_INFO_LVL_5,
                                            MCA_BASE_VAR_SCOPE_GROUP, &mca_osc_ucx_component.acc_single_intrinsic);
+    (void) mca_base_component_var_register(&mca_osc_ucx_component.super.osc_version, "enable_nonblocking_accumulate",
+                                           description_str, MCA_BASE_VAR_TYPE_BOOL, NULL, 0, 0, OPAL_INFO_LVL_5,
+                                           MCA_BASE_VAR_SCOPE_GROUP, &enable_nonblocking_accumulate);
+    (void) mca_base_component_var_register(&mca_osc_ucx_component.super.osc_version, "enable_wpool_thread_multiple",
+                                           description_str, MCA_BASE_VAR_TYPE_BOOL, NULL, 0, 0, OPAL_INFO_LVL_5,
+                                           MCA_BASE_VAR_SCOPE_GROUP, &mpi_thread_multiple_enabled);
```

Comment:
Answered, please see above.

Reply:
I'm still not sure why this is an MCA parameter? Why would a user have to control thread-safety from the command line?

Reply:
With the latest commits, we are setting this variable as the same as the opal_thread_multiple(). 
I would like to keep this as an MCA variable for the following reason: Users will have the option to provide each window with an endpoint, instead of sharing it across all the windows. So far, I have not found such an application, but it would be harmless to keep this option open.  What do you think?